conductor/build.gradle

```diff
@@ -17,29 +17,13 @@ android {
     }
 }
 
-configurations {
-    lintChecks
-}
-
 dependencies {
     testImplementation rootProject.ext.junit
     testImplementation rootProject.ext.roboelectric
 
     api rootProject.ext.androidxAnnotations
 
-    lintChecks project(path: ':conductor-lint', configuration: 'lintChecks')
-}
-
-task copyLintJar(type: Copy) {
-    from(configurations.lintChecks) {
-        rename { 'lint.jar' }
-    }
-    into 'build/intermediates/lint/'
-}
-
-project.afterEvaluate {
-    def compileLintTask = project.tasks.find { it.name == 'compileLint' }
-    compileLintTask.dependsOn(copyLintJar)
+    lintChecks project(':conductor-lint')
```

Comment:
You want `lintPublish` here (added in 3.4, which is why those 3.0 docs don't have it).

`lintChecks` = "use these lint checks to validate my project"

`lintPublish` = "use these lint checks to validate my project and projects dependent on my project"